exec/plugin-commands-rebuild/test/recursive.ts

```diff
@@ -148,12 +148,11 @@ test('pnpm recursive rebuild with hoisted node linker', async () => {
   await projects['project-4'].has('@pnpm.e2e/pre-and-postinstall-scripts-example/generated-by-postinstall.js')
 })
 
-// TODO: make this test pass
-test.skip('rebuild multiple packages in correct order', async () => {
+test('rebuild multiple packages in correct order', async () => {
```

Comment:
This test seems to be passing after the `pnpm-workspace.yaml` fix below. I'm guessing this was originally flaking due to `json-append` race conditions as well and can now be re-enabled.